src/borg/compress.pyx

```diff
@@ -167,7 +170,10 @@ class ZLIB(CompressorBase):
 
     def decompress(self, data):
         # note: for compatibility no super call, do not strip ID bytes
-        return zlib.decompress(data)
+        try:
+            return zlib.decompress(data)
+        except zlib.error as e:
+            raise DecompressionError(str(e)) from None
```

Comment:
This doesn't work correctly, the docstring / "excstring" of DecompressionError needs a placeholder, like so:

`"Failed to decompress chunk: {}"`